### DISCUSSION TITLE
setting up jaeger UI locally

### DISCUSSION THREAD

**Discussion by AdityaRaimec22** [jaegertracing/jaeger#4632] [Q&A]
when I run this command  yarn install --frozen-lockfile  I gets 
` yarn install --frozen-lockfile`
`Usage: yarn [options]
yarn: error: no such option: --frozen-lockfile`

and it is written If you don't have it running locally, then tunnel to the correct host and port:

`ssh -fN -L 16686:$BACKEND_HOST:$BACKEND_PORT $BACKEND_HOST`

but I am unable to figure out the BACKEND_PORT and BACKEND_HOST

**Comment by psk001** [comment]
Did you go to jaeger-ui directory after cloning the repo?
For second part of your problem. 
Set up jaeger all in one docker first. [Here ](https://www.jaegertracing.io/docs/1.6/getting-started/) 

This will expose required ports such as 16686.

**Comment by yurishkuro** [comment]
> `Usage: yarn [options] yarn: error: no such option: --frozen-lockfile`

make sure you run the latest version of `yarn`
